content/posts/2022/introducing-launch-on-reana.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Introducing Launch on REANA"
+date: 2022-04-20T14:00:00+02:00
+tags:
+  - API
+  - cluster
+---
+
+We are happy to announce that it is now possible to launch workflows that are
+hosted on external services directly from the REANA web interface, without
+relying on the `reana-client` CLI tool!
+
+<!--more-->
+
+## Running workflows using the command-line client
+
+As you might already know, `reana-client` is the command-line tool that can be
+used to run workflows on REANA. Up until now, using this tool was the only way
+to create workflows, to upload files to their workspaces and to execute them. A
```

Comment:
*suggestion:* Technically command-line tool is not the only way to use REANA. We also support Python API - https://blog.reana.io/posts/2021/reana-client-python-api/. Would be good to link this blog post here for better SEO.